testsuite/forge-cli/src/main.rs

```diff
@@ -1773,7 +1773,15 @@ fn realistic_env_max_load_test(
                     mempool_backlog: 40000,
                 })
                 .init_gas_price_multiplier(20),
-            inner_success_criteria: SuccessCriteria::new(if ha_proxy { 4600 } else { 6800 }),
+            inner_success_criteria: SuccessCriteria::new(
+                if ha_proxy {
+                    4600
```

Comment:
Any reason ha proxy is so low as compared to others?

Reply:
I didn't touch ha proxy. That test actually needs investigation -- I suspect network emulation is not working for it...

Reply:
My guess is that when HAProxy was setup, it wasn't properly configured. IIRC, folks just chose some numbers and ran 😄